File: 알고리즘/2/Thousand Separator.py
class Solution:
    def thousandSeparator(self, n: int) -> str:
        i=0 
        result =''
        if(n==0):
            return "0"
        
        while n>0:
            if i%3==0 and i!=0:
                result = '.'+result
            
            result = str(n%10) + result
            n//=10
            i+=1
            
        return result
            

File: 알고리즘/2/Thousand Separator.cpp
#include <bits/stdc++.h>

using namespace std;

class Solution {
public:
    string thousandSeparator(int n) {
        if(n==0) return "0";
        int i = 0;

        string result ="";
        for(int i=0; n>0; i++){
          if(i%3==0 && i!=0){
            result = "." +result;
          }
          result += to_string(n%10);
          n/=10;
        }
        return result;
    }
};